src/mesido/workflows/grow_workflow.py

```diff
@@ -133,7 +133,8 @@ class EndScenarioSizing(
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-        self.heat_network_settings["minimum_velocity"] = 0.0  # 0.001
+        self.heat_network_settings["minimum_velocity"] = 1.0e-4
```

Comment:
Add a comment why this value is picked, above this line of code.

Reply:
Nice for picking this up. I added a comment